doc/source/data/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -439,7 +439,7 @@
    "source": [
     "Then we use the {meth}`~ray.data.Dataset.map_batches` API to apply the model to the whole dataset.\n",
     "\n",
-    "The first parameter of `map_batches` is the user-defined function (UDF), which can either be a function or a class. Since we are using a class in this case, the UDF will run as long-running [Ray actors](actor-guide). For class-based UDFs, we use the `compute` argument to specify {class}`~ray.data.ActorPoolStrategy` with the number of parallel actors.\n",
+    "The first parameter of `map_batches` is the user-defined function (UDF), which can either be a function or a class. Since we are using a class in this case, the UDF will run as long-running [Ray actors](actor-guide). For class-based UDFs, we use the `concurrency` argument to specify the number of parallel actors.\n",
```

Comment:
```suggestion
    "The first parameter of `map_batches` is the user-defined function (UDF), which can either be a function or a class. Because this case uses a class, the UDF runs as long-running [Ray actors](actor-guide). For class-based UDFs, use the `concurrency` argument to specify the number of parallel actors.\n",
```

Reply:
updated.